packages/bruno-app/src/components/Environments/EnvironmentSettings/EnvironmentList/EnvironmentDetails/EnvironmentVariables/index.js

```diff
@@ -34,24 +41,46 @@ const EnvironmentVariables = ({ environment, collection }) => {
 
   const handleVarChange = (e, _variable, type) => {
     const variable = cloneDeep(_variable);
+    let newValidationErrors = { ...validationErrors };
+
     switch (type) {
       case 'name': {
-        variable.name = e.target.value;
+        const newName = e.target.value;
+
+        // Perform validation for the name (e.g., check if it's not empty)
+        if (!newName.trim()) {
```

Comment:
@razzivofficial We need to the validation for environment variable. At the moment we will default to the naming restrictions of javascript lang itself

*Variable Names Must Start with a Letter, Underscore (_), or Dollar Sign ($)*

This needs to be covered right ?